### ISSUE TITLE
[BUG] API is not being served by the app gateway

### ISSUE THREAD

**Issue by TessFerrandez** [microsoft/AzureTRE#100]
**Describe the bug**

API is not being served by the app gateway

**Steps to reproduce**

1. Deploy the API using the instructions in docs/quickstart.md
1. Access the API from the app gateway

API is not being served

**Comment by christoferlof** [comment]
My findings
Seeing error from app gw health probe
![image](https://user-images.githubusercontent.com/166592/119365999-cdec1800-bcb0-11eb-99a7-d24a77c47f33.png)

No custom probe configured for management api http setting
![image](https://user-images.githubusercontent.com/166592/119366419-3e933480-bcb1-11eb-9038-cd1adebafed8.png)

Updated custom probe config to use the probe we have created pointing to /api/ping.
![image](https://user-images.githubusercontent.com/166592/119366825-a6497f80-bcb1-11eb-8e6e-844d7db77149.png)

Testing the probe config, results in 200 ok
![image](https://user-images.githubusercontent.com/166592/119366770-96ca3680-bcb1-11eb-927a-990ade8fe761.png)

api/ping returns 200 ok through app gw
![image](https://user-images.githubusercontent.com/166592/119366940-c2e5b780-bcb1-11eb-8373-9a28525eb8a5.png)

Bottom line - probe config is missing on http config
